eng/Versions.props

```diff
@@ -316,7 +316,7 @@
       When updating the S.C.I or S.R.M version please let the MSBuild team know in advance so they
       can update to the same version. Version changes require a VS test insertion for validation.
     -->
-    <SystemCollectionsImmutableVersion>7.0.0</SystemCollectionsImmutableVersion>
+    <SystemCollectionsImmutableVersion>8.0.0</SystemCollectionsImmutableVersion>
```

Comment:
I'm not 100% certain we can do this - what triggered the upgrade here?  If VS 17.8 itself isn't on 8.0 I believe we'll have issues

Reply:
MSBuildStructuredLoggerVersion version 2.2.386 depends on this apparently. Maybe we can try with a lower MSBuildStructuredLoggerVersion?

Reply:
@jordi1215 yeah lets try a lower version.  Try 2.2.350, if that doesn't work I can help get other versions added so restore works